persist-cli/src/main/java/io/ballerina/persist/cmd/Init.java

```diff
@@ -113,88 +125,53 @@ public void execute() {
                     + e.getMessage());
             return;
         }
-
+        if (schemaFiles.size() > 1) {
+            errStream.println("ERROR: persist directory contains multiple schema files. ");
+            return;
+        }
         String packageName;
         try {
             packageName = readPackageName(this.sourcePath);
         } catch (BalException e) {
             errStream.println(e.getMessage());
             return;
         }
+        if (module == null) {
+            module = packageName;
+        } else {
+            module = module.replaceAll("\"", "");
+        }
         if (schemaFiles.size() == 0) {
-            schemaFiles.add(packageName);
+            schemaFiles.add(SCHEMA_FILE_NAME);
```

Comment:
why do we need to add to the files list?

Reply:
I thought of keeping the list as multiple file support will be added later. shall I remove it?

Reply:
removed in [566e36d](https://github.com/ballerina-platform/persist-tools/pull/148/commits/566e36dd84d453026fbf2506266f34668c08e1af)